docs/azure-setup.md

```diff
@@ -31,25 +31,23 @@ iotedgedev iothub setup
 
 You can use the following `az cli` command to create a service principal:
 
-```
+```sh
 az ad sp create-for-rbac -n "iotedgedev01"
 ```
 
 > Note: Running `iotedgedev iothub setup` without any other parameters will save you time from looking up the required parameter values. The command will help you choose the parameters in an interactive way.
 
-Alternatively, you can deploy the IoT Hub **and** Container Registry with this **Deploy to Azure** template:
-
-[![Azure Deployment](https://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fiotedgedev%2Fmaster%2Fassets%2Fdeploy%2FARMDeployment%2Fazuredeploy.json)
```

Comment:
Neither of these links work, but I only removed the image link for now: https://azuredeploy.net/deploybutton.png